DVR/SessionDataTask.swift

```diff
@@ -30,67 +30,70 @@ class SessionDataTask: NSURLSessionDataTask {
     }
 
     override func resume() {
-        let cassette = session.cassette
+        let queue = dispatch_queue_create("com.venmokit.DVR.sessionDataTaskQueue", nil)
+        dispatch_async(queue) { [unowned self] in
```

Comment:
I stopped to wonder if we should call this all in this queue or just call the completion callback in this queue. I think both cases are valid and both accomplish the same thing, but putting all this in a block seems like a bigger hammer than needed, but probably simulates closer to what NSURLSession might be doing.

I'm curious what might be best!